core/App/screens/Chat.tsx

```diff
@@ -39,7 +39,8 @@ const Chat: React.FC<ChatProps> = ({ navigation, route }) => {
         _id: m.id,
         text: m.content,
         record: m,
-        createdAt: m.createdAt,
+        createdAt: m.sentTime,
```

Comment:
Does this impact the out of order chat messages? Was this intentional?

Reply:
I was hoping it would impact the out of order messages, but I wasn't able to replicate so I couldn't verify. it doesn't seem to hurt anything at least